common/iverilog.py

```diff
@@ -0,0 +1,10 @@
+import os
```

Comment:
I propose we consolidate this and `common/irun.py` into a single file, say called, `compile_verilog.py`. This file should have 3 methods:
- `irun(files, ...)`
- `iverilog(files)`
- `compile_verilog_or_die(files, **kwargs)`

The first two functions can be copy and paste of the originals. The last function will look as follows:

```
def compile_verilog_or_die(files, **kwargs):
    if irun_available():
        return irun(files, **kwargs)
    if iverilog_available():
        return iverilog(files, **kwargs)
    raise Exception("...")
```
Note sure if `compile` is the right term here, but basically it would be nice to think of this as there being an interface to "compiling" or "running" verilog (or whatever it should be called); `irun` and `iverilog` are 2 separate implementations of this.